### PR TITLE
Update frontpage and 12/ page

### PR DESCRIPTION
**Pull request by darcywong00**
* Change beta links to stable
* Update list of what's new

features pages
* Update references to Unicode 12.0

Also fix a link to `#bcp47-tags` (#34)

### CODE REVIEW

File: 12/index.php [hunk 1]
```diff
@@ -7,51 +7,50 @@
...
         'background' => 'water'
     ]
 ]);
 ?>
 
-<h2>Announcing the release of Keyman 12.0 Beta</h2>
-<p class="red">7 August 2019</p>
-<p>We are pleased to announce that Keyman 12.0 is now in beta!</p>
+<h1>Keyman 12.0 Stable</h1>
+<p class="red">6 October 2019</p>
```
Comment (mcdurdin) [inline 12/index.php]: Technically 7 Oct ;-)

File: desktop/features.php [hunk 2]
```diff
@@ -344,9 +344,9 @@
         <img src="<?php echo cdn("img/developer9.png"); ?>"/>
     </div>
     <div class="section" id="unicode">
-        <h2 class="red underline">Unicode 11.0 Compliant</h2>
+        <h2 class="red underline">Unicode 12.0 Compliant</h2>
```
Comment (mcdurdin) [inline desktop/features.php]: Can I squirm here? Not sure we have updated the unicode character database to 12.0...

Comment (darcywong00) [reply desktop/features.php]: PR keymanapp/keyman#1834 added the Unicode 12.0 characters.

Comment (mcdurdin) [reply desktop/features.php]: Well there ya go. I approved that PR and all. 😁

**Pull request merged**